emrun.py

```diff
@@ -704,6 +704,8 @@ def do_POST(self):
         if not emrun_options.serve_after_exit:
           page_exit_code = int(data[6:])
           logv('Web page has quit with a call to exit() with return code ' + str(page_exit_code) + '. Shutting down web server. Pass --serve-after-exit to keep serving even after the page terminates with exit().')
+          if emrun_options.force_exit:
+            self.server.socket.setblocking(False)
```

Comment:
Is there a downside to always doing this? I mean, instead of adding an option, to always set nonblocking after the page exits?

Reply:
From our side there is no downside specific to this change.  We just thought that it might be categorically better received if it were presented as an option, but there is no downside to always doing this before shutdown.

Reply:
Ok, then I think it's better to avoid adding an option actually. That way it will be fully tested.